sky/client/sdk.py

```diff
@@ -212,13 +213,23 @@ def list_accelerator_counts(
 @annotations.client_api
 def optimize(
     dag: 'sky.Dag',
-    minimize: common.OptimizeTarget = common.OptimizeTarget.COST
+    minimize: common.OptimizeTarget = common.OptimizeTarget.COST,
+    cluster_name: Optional[str] = None,
+    idle_minutes_to_autostop: Optional[int] = None,
+    down: bool = False,  # pylint: disable=redefined-outer-name
+    dryrun: bool = False
```

Comment:
Should we emphasize that these are only required when `admin_policy` is specified, or more aggressively we just make this:
```suggestion
    admin_policy_request_options: Optional[admin_policy.RequestOptions] = None
```

Reply:
Yep good idea, I was also thinking about it but wasn't sure if we want to expose `admin_policy.RequestOptions` in the SDK to end users. Should be ok.